nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -223,13 +224,13 @@ protected String defaultWarehouseLocation(TableIdentifier table) {
 
   @Override
   public List<TableIdentifier> listTables(Namespace namespace) {
-    return client.listTables(namespace);
+    return refreshedClient().listTables(namespace);
```

Comment:
refreshing the client is enough for a few APIs that doesn't use `TableOperations`. But the code looks odd. So, I am refreshing all the APIs as it is a lightweight operation.